### ISSUE TITLE
Server Connection Issues

### ISSUE THREAD

**Issue by baker2gs** [robertovg/audiostation-steroids#5]
Hi, I'm getting an error:

Sorry, we have detect problems with our servers. Please try again later!

![image](https://user-images.githubusercontent.com/24899325/31793773-d09e47a2-b4e5-11e7-99f8-f9ca304118df.png)

Thanks for the kick ass extension.

**Comment by robertovg** [comment]
Yes, it's because the server is down, but I made the decision of turning it off completely. As I wrote in the description of the project now, I encourage anyone interested in continuing the project 👨‍💻. Sorry, and I hope you have an alternative or even want to continue with it 💪.
